compiler/front/options.nim

```diff
@@ -1162,16 +1164,12 @@ proc canonicalCase(path: var string) =
   else: toLowerAscii(path)
 
 proc fileInfoKnown*(conf: ConfigRef; filename: AbsoluteFile): bool =
-  var
-    canon: AbsoluteFile
-  try:
-    canon = canonicalizePath(conf, filename)
-  except OSError:
-    canon = filename
-  canon.string.canonicalCase
-  result = conf.m.filenameToIndexTbl.hasKey(canon.string)
+  result = conf.fileIdxTbl.hasKey(filename.string)
```

Comment:
This is not correct, as it means that `fileInfoKnown(x)` would return 'false' if there's no preceding call to `fileInfoIdx(x)`.

Reply:
this proc is only used in nimsuggest api, I assuming the project already compiled and  `fileInfoIdx` called on every project files, no?

Reply:
am not sure now, just revert it.